neo4j-to-neptune/src/main/java/com/amazonaws/services/neptune/util/Utils.java

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2025 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License").
+You may not use this file except in compliance with the License.
+A copy of the License is located at
+    http://www.apache.org/licenses/LICENSE-2.0
+or in the "license" file accompanying this file. This file is distributed
+on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied. See the License for the specific language governing
+permissions and limitations under the License.
+*/
+
+package com.amazonaws.services.neptune.util;
+
+public class Utils {
```

Comment:
Nit: add private constructor to prevent instantiation of utility class
```suggestion
public class Utils {
    private Utils() {}
```